src/components/home/proposed/ProposedDongilItem.tsx

```diff
@@ -50,13 +55,27 @@ function ProposedDongilItem({
       totalPrice: totalPrice,
       weekPrice: weekPrice,
       weeks: weeks,
-      filename: 'dummy',
+      fileName: fileName,
       shouldCloseOnOverlayClick: true,
     });
   }
 
+  const openNoticeSundaySheet = () => {
+    setOpenBottomSheet({
+      sheetContent: 'Notice',
+      sheetProps: {
+        open: true,
+      },
+      contentProps: {
+        type: 'sunday',
+      },
+    });
+  };
+
   return (
-    <StyledButton onClick={openProposedReceiptModal}>
+    <StyledButton
+      onClick={now === 7 ? openNoticeSundaySheet : openProposedReceiptModal}
```

Comment:
구체적이지 않은 변수명과 매직넘버가 혼용되어 가독성이 떨어지는 코드 같습니다.

Reply:
위와 같습니다. 
일요일은 항상 7 오히려 더 직관적이라고 봄.

Reply:
알겠습니당!!

Reply:
코드 작성자는 직관적일 수 있지만 처음 이 코드를 읽는 사람입장에서 생각하면 저라면,
* now -> currentDayOfTheWeek
* 7 -> SUNDAY (macro or enum 사용)
```suggestion
      const SUNDAY = 7;
      onClick={currentDayOfWeek === SUNDAY ? openNoticeSundaySheet : openProposedReceiptModal}
```
로 작성할것 같습니다.

Reply:
이런건 이넘 만들어두면 다른곳에서 재사용할 수도 있구요